eventGenerator/wind.go

```diff
@@ -45,6 +45,9 @@ func GenerateWindEvent() {
 		return
 	}
 	var windPower float64 = math.Round(normPower(mean)*10) / 10
+	if windPower < 1 {
+		windPower = 20.0
```

Comment:
No calm?

Reply:
what do you mean?

Reply:
I mean that windPower can not be zero, according to your code. And also maybe `math.Abs` the value

Reply:
i measure power in km per hour, so naturally, it cant be less than 1 and i dont generate negative values so no need to get absolute value